WordPress/src/main/java/org/wordpress/android/ui/prefs/DetailListPreference.java

```diff
@@ -226,20 +217,12 @@ public View getView(final int position, View convertView, ViewGroup parent) {
 
             if (mainText != null && getEntries() != null && position < getEntries().length) {
                 mainText.setText(getEntries()[position]);
-                mainText.setTypeface(TypefaceCache.getTypeface(getContext(),
-                        TypefaceCache.FAMILY_DEFAULT,
-                        Typeface.NORMAL,
-                        TypefaceCache.VARIATION_NORMAL));
             }
 
             if (detailText != null) {
                 if (mDetails != null && position < mDetails.length && !TextUtils.isEmpty(mDetails[position])) {
                     detailText.setVisibility(View.VISIBLE);
                     detailText.setText(mDetails[position]);
-                    detailText.setTypeface(TypefaceCache.getTypeface(getContext(),
```

Comment:
Man, visual diffs really show the code bloat! Didn't realize how many times I did the exact same thing in this file...